[2022-06-27 07:24:08,763] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: test.accurate manual__2022-06-27T07:24:05.517943+00:00 [queued]>
[2022-06-27 07:24:08,771] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: test.accurate manual__2022-06-27T07:24:05.517943+00:00 [queued]>
[2022-06-27 07:24:08,771] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 07:24:08,771] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-06-27 07:24:08,771] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 07:24:08,781] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): accurate> on 2022-06-27 07:24:05.517943+00:00
[2022-06-27 07:24:08,786] {standard_task_runner.py:52} INFO - Started process 1944 to run task
[2022-06-27 07:24:08,789] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test', 'accurate', 'manual__2022-06-27T07:24:05.517943+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/second_2.py', '--cfg-path', '/tmp/tmpzetomv7q', '--error-file', '/tmp/tmpvxqrk5o5']
[2022-06-27 07:24:08,789] {standard_task_runner.py:80} INFO - Job 97: Subtask accurate
[2022-06-27 07:24:08,827] {logging_mixin.py:109} INFO - Running <TaskInstance: test.accurate manual__2022-06-27T07:24:05.517943+00:00 [running]> on host f32f6d4577d5
[2022-06-27 07:24:08,867] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=accurate
AIRFLOW_CTX_EXECUTION_DATE=2022-06-27T07:24:05.517943+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-27T07:24:05.517943+00:00
[2022-06-27 07:24:08,868] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-27 07:24:08,868] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'accurate'"]
[2022-06-27 07:24:08,879] {subprocess.py:85} INFO - Output:
[2022-06-27 07:24:08,881] {subprocess.py:89} INFO - accurate
[2022-06-27 07:24:08,881] {subprocess.py:93} INFO - Command exited with return code 0
[2022-06-27 07:24:08,899] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=test, task_id=accurate, execution_date=20220627T072405, start_date=20220627T072408, end_date=20220627T072408
[2022-06-27 07:24:08,920] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-27 07:24:08,939] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
